docs/src/pages/examples/rag/filter-rag.mdx

```diff
@@ -100,7 +104,13 @@ const chunkMetadata = chunks?.map((chunk: any, index: number) => ({
 
 ## Agent Configuration
 
-Set up the Mastra agent that will handle the responses:
+The agent is configured to understand user queries and translate them into appropriate metadata filters.
+
+In our instructions we do the following:
+- Provide the agent with the metadata structure so that it knows the fields available for filtering.
+- Provide a prompt (e.g. PGVECTOR_PROMPT) that provides the agent with filter operations and syntax for the vector store queries.
+
+> Note: Different vector stores have specific prompts available. See [Vector Store Prompts](../../docs/reference/tools/vector-query-tool#vector-store-prompts) for details.
```

Comment:
Same here. We might want to point this to a different location

Reply:
Maybe put the code sample back in the retrieval section with createVectorQueryTool? With the Tab Switcher?

Reply:
This text feels like it belongs these rather than in the example

Reply:
Updated the text in the example and changed the location of the note